decentraland/unity-explorer
# Feat: analytics badges activity events (height, steps)

## What does this PR change?

- implements **Height Reached** event 
- refactored **Walked Distance** event to report steps instead of distance
- add flush and send analytics events on application quit
- removed duplication on Avatar animation events handling 

## How to test the changes?

1. Launch the explorer
2. Verify that step/jump/land SFXs and particles are same as on prod (and happens at the same animation time events)
3. Verify "happy path" is not broken

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
